static/app/views/automations/components/automationBuilder.tsx

```diff
@@ -21,6 +24,12 @@ import {
 
 export default function AutomationBuilder() {
   const {state, actions} = useAutomationBuilderContext();
+  const organization = useOrganization();
+  const api = useApi();
+
+  useEffect(() => {
+    fetchOrgMembers(api, organization.slug);
+  }, [api, organization]);
```

Comment:
we need this for the [SelectMembers](https://github.com/getsentry/sentry/blob/master/static/app/components/selectMembers/index.tsx) component to properly load

Reply:
I originally had this in the email action component, but then I realized that this would be called for each instance of the component vs. having this at the top-level within the automation builder, where it should only get called once. We will also need the org members data for one of the data condition nodes (screenshot below). Let me know if you still think it should be within the component!

<img width="606" alt="Screenshot 2025-05-15 at 10 01 07 AM" src="https://github.com/user-attachments/assets/379e80ca-9815-4f60-a8d8-39ff2e428e2d" />

Reply:
I still think it should be moved down to the component! Ideally this fetch call should be deduplicated by taking advantage of React Query, but that probably needs to be done in a separate PR.

Reply:
For now I'd just add an inline comment explaining why this is here